cli/internal/codegen/testdata/expected_typescript.ts

```diff
@@ -375,7 +389,7 @@ class BaseClient {
     }
 
     // callAPI is used by each generated API method to actually make the request
-    public async callAPI(method: string, path: string, body?: BodyInit, params?: CallParameters): Promise<Response> {
+    public async callAPI(method: string, path: string, requiresAuth: boolean, body?: BodyInit, params?: CallParameters): Promise<Response> {
```

Comment:
Should this be public? If we're changing the signature it suggests it's not stable and should therefore not be public?

Reply:
The base client is private and can’t be accessed access of this file